docs/pages/versions/unversioned/sdk/speech.md

```diff
@@ -18,7 +18,7 @@ Speak out loud the `text` given `options`. Calling this when another text is bei
 -   **options : `object`** --
 
       A map of options:
-
+    -   **voiceIOS : `string`** -- Voice identifier (iOS only)
```

Comment:
Since this parameter was not documented, can we remove `IOS` suffix?
```suggestion
    -   **voice : `string`** -- Voice identifier (**iOS only**)
```